models/message.go

```diff
@@ -108,25 +108,20 @@ func CreateMessage(fields *Message) error {
 // URLs, and then passing the body through Blackfriday for additional parsing
 // and finally Bluemonday for sanitization.
 func ParseMessage(message *Message) string {
-	body := message.Body
+	messageCopy := *message
```

Comment:
@jmingtan could you explain more why you want to use a pointer here? And I think we probably want to avoid mutating the message's internal state -- why not just return a new value for `body` that goes into the response?

(Which is to say, I think I agree with you, @whatupdave, but I don't want to put words in your mouth.)